language/tools/move-cli/src/main.rs

```diff
@@ -0,0 +1,440 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use errmapgen::ErrorMapping;
+use move_cli::OnDiskStateView;
+use move_core_types::{
+    account_address::AccountAddress,
+    gas_schedule::{GasAlgebra, GasUnits},
+    language_storage::TypeTag,
+    parser,
+    transaction_argument::TransactionArgument,
+    vm_status::{AbortLocation, StatusCode, VMStatus},
+};
+use move_lang::{self, compiled_unit::CompiledUnit, shared::Address};
+use move_vm_runtime::{data_cache::TransactionEffects, move_vm::MoveVM};
+use move_vm_types::{gas_schedule, values::Value};
+use vm::{
+    errors::VMError,
+    file_format::{CompiledModule, CompiledScript},
+};
+
+use anyhow::{anyhow, bail, Result};
+use std::{fs, path::Path};
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+#[structopt(name = "Move", about = "CLI frontend for Move compiler and VM")]
+struct Move {
+    /// Directory storing Move resources and module bytecodes produced by script execution.
+    #[structopt(name = "data", long = "data", default_value = "move_data")]
+    move_data: String,
+    /// Directory storing Move source files that will be compiled and loaded into the VM before
+    /// script execution.
+    #[structopt(name = "lib", long = "lib", default_value = "move_lib")]
```

Comment:
We should try to keep these args the same across the various tools. Tagging in @bob-wilson since I think he looked at that in the past